opennms-webapp/src/main/java/org/opennms/web/event/filter/NegativeSystemIdFilter.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2002-2016 The OpenNMS Group, Inc.
```

Comment:
The year here must be the year when the file was created. In this case 2016.
The line below must always be 1999-<current-year>.